docs/core-concepts.md

```diff
@@ -75,16 +75,35 @@ const context = await browser.newContext({
 
 <br/>
 
-## Pages and frames
+## Pages, frames and navigation
```

Comment:
I'd leave as is. User will not pick this because they were interested in 'navigation'.